server.rb

```diff
@@ -93,9 +93,15 @@ def html(view)
   end
 
   def sort_versions(ary)
-    valid_version_numbers = ary.select { |i| i if i.match(/(0-9|\.|\-)/)}
-    non_valid_version_numbers = ary - valid_version_numbers
-    (valid_version_numbers.sort_by {|v| Gem::Version.new( v.gsub(/^[a-z|A-Z|.]*/, '') ) } + non_valid_version_numbers)
+    valid_version_numbers = ary.select { |i| i if i.match(/^[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+(-[[:alnum:]]+)?$/) }
+    non_valid_version_numbers = (ary - valid_version_numbers).sort
+    versions = valid_version_numbers.sort_by {|v| Gem::Version.new( v.gsub(/^[a-z|A-Z|.]*/, '') ) } + non_valid_version_numbers
+    if versions.include?('latest')
```

Comment:
Are you talking about the block to move latest to the top? I did that because locally we have development containers that are newer then our 'latest' container, which is our stable build. I can remove that.